tests/integration_tests/test_mlflow.py

```diff
@@ -0,0 +1,98 @@
+import py
+import unittest
+from unittest.mock import patch
+
+import optuna
+from optuna.integration.mlflow import MlflowCallback
+from tests.test_structs import _create_frozen_trial
+
+
+def test_happy_case(tmpdir):
+    # type: (py.path.local) -> None
+
+    db_file_name = "file:{}".format(tmpdir)
+
+    mlflc = MlflowCallback(
+        tracking_uri=db_file_name, metric_name="my_metric", experiment="my_experiment"
+    )
+    study = optuna.create_study()
+    ft = _create_frozen_trial()
+    mlflc(study, ft)
```

Comment:
In addition to unit testing just for the function, how about adding more end-to-end cases where the callback is invoked via the `optimize` method? I think it improves coverages in terms of practical scenarios.

One of the simplest cases would be just like what we test in the doctest.

```python
def objective(trial):
    x = trial.suggest_uniform('x', -10, 10)
    return (x - 2) ** 2

tempdir = tempfile.mkdtemp()
uri = pathlib.Path().as_uri()

mlflc = MlflowCallback(tracking_uri=uri)
study = optuna.create_study(study_name='my_study')
study.optimize(objective, n_trials=10, callbacks=[mlflc])

# Assert mlflow data.
...
```

Reply:
I implemented an end to end test.